helpers/formValidation.tsx

```diff
@@ -1,107 +1,128 @@
 import * as Yup from 'yup'
 
-const TEXT_MIN = 2
-const TEXT_MAX = 64
-const PASSWORD_MIN = 6
-const REGEX_ALPHANUMERICS_AND_SPACE = /^[a-zA-Z0-9_\s]*$/
+const Regex = {
```

Comment:
It wasn't needed to make it an object but made for consistency.

Reply:
Unless there is high potential for there to be more regexes used in this file in the future, then this is fine. But if this is the only regex we can foresee us using, then this constant should be declared directly as a regex rather than an object